assets/airports/eidw.json

```diff
@@ -1009,316 +1439,398 @@
-            "altitude": null,
-            "speed": null,
+            "altitude": "",
+            "speed": "",
             "method": "random",
-            "rate": 3.5,
+            "rate": 1,
             "airlines": [
-                ["ein", 12],
-                ["ein/long", 3],
-                ["ryr", 10],
-                ["stk",  7],
-                ["baw",  5],
-                ["bcy",  5],
-                ["bee",  3],
-                ["aal/long",  3],
-                ["aca/long",  2],
-                ["dal/long",  3],
-                ["ual/long",  3],
-                ["swr",  1],
-                ["ibe",  1]
+                ["aal/long", 5],
+                ["aca/long", 3],
+                ["bee",      5],
+                ["dal/long", 3],
+                ["ein",     20],
+                ["ein/long", 5],
+                ["eth/long", 3],
+                ["fastga",   3],
+                ["nax",      3],
+                ["ryr",     30],
+                ["sas",      5],
+                ["stk",     15],
+                ["ual/long", 3]
             ]
         },
         {
             "origin": "EIDW",
             "destination": "",
             "category": "departure",
             "route": "EIDW.PESIT.PESIT",
-            "altitude": null,
-            "speed": null,
+            "altitude": "",
+            "speed": "",
             "method": "random",
-            "rate": 3.5,
+            "rate": 4,
             "airlines": [
-                ["ein", 12],
-                ["ein/long", 3],
-                ["ryr", 10],
-                ["stk",  7],
-                ["baw",  5],
-                ["bcy",  5],
-                ["bee",  3],
-                ["aal/long",  3],
-                ["aca/long",  2],
-                ["dal/long",  3],
-                ["ual/long",  3],
-                ["swr",  1],
-                ["ibe",  1]
+                ["aea",    1],
+                ["aee",    1],
+                ["afr",    3],
+                ["baw",   10],
+                ["bcy",   10],
+                ["bee",    5],
+                ["ein",   20],
+                ["fastga", 3],
+                ["fpo",    1],
+                ["ibs",    5],
+                ["ryr",   30],
+                ["stk",   15],
+                ["swr",    3],
+                ["tay",    1],
+                ["thy",    4],
+                ["tvf",    2],
+                ["tvs",    1],
+                ["vlg",    2]
             ]
         },
         {
             "origin": "",
             "destination": "EIDW",
             "category": "arrival",
-            "route": "_EIDW305060..OLAPO..RONON..ORVEN..GIRAS..KERAV..KOGAX..KUDOM..DW814..DW815..DW816..NARMU..LAPMO",
-            "altitude": 10000,
-            "speed": 250,
-            "method": "surge",
-            "rate": 10,
-            "entrail": [10, 35],
+            "route": "ERNAN.OLAPO.EIDW",
+            "altitude": 34000,
+            "speed": 350,
```

Comment:
no speeds higher than 320 please. If/when the game is able to separate airplanes later on, we can do this sort of thing, but since they all slow down at different rates, we can have some undesirable compression effects between aircraft, and they could end up arriving at the airspace without adequate separation, for which the user will lose points. So when it's all turbojet aircraft, we can go up to 320, but otherwise `280` is the norm, just for those reasons.